src/app/user/[_id]/submit-request/page.js

```diff
@@ -1,12 +1,82 @@
 "use client";
+import { useState } from 'react';
+import Button from 'react-bootstrap/Button';
+import Col from 'react-bootstrap/Col';
+import Form from 'react-bootstrap/Form';
+import InputGroup from 'react-bootstrap/InputGroup';
+import Row from 'react-bootstrap/Row';
+
+function FormExample() {
+  const [validated, setValidated] = useState(false);
+
+  const handleSubmit = (event) => {
+    const form = event.currentTarget;
+    if (form.checkValidity() === false) {
+      event.preventDefault();
+      event.stopPropagation();
+    }
+
+    setValidated(true);
+  };
 
-const page = () => {
   return (
-    <div>
-    <h2>This page file is for the submission form that a resident user can submit.</h2>
-    <h3>This should be wired up to the landing page and even the nav bar in the footer with some of Submit Request button.</h3>
-    </div>
-  )
+    <Form noValidate validated={validated} onSubmit={handleSubmit}>
+      <Row className="mb-3">
+        <Form.Group as={Col} md="4" controlId="validationCustom01">
+          <Form.Label>First name</Form.Label>
+          <Form.Control
+            required
+            type="text"
+            placeholder="First name"
+            defaultValue="Mark"
+          />
+          <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
```

Comment:
I don't think this line is necessary. You would really only want something like this when there is an issue with one of the inputs. The same can be said about line 43.